flow/layers/physical_shape_layer.h

```diff
@@ -45,6 +52,11 @@ class PhysicalShapeLayer : public ContainerLayer {
   bool isRect_;
   SkRRect frameRRect_;
   Clip clip_behavior_;
+
+#if defined(OS_FUCHSIA)
+  std::unique_ptr<scenic::EntityNode> entity_node_ptr_;
+  mutable bool fuchsia_painted_ = false;
```

Comment:
I'm not sure I understand this variable, could you add a comment?

Reply:
Done.